docs/shared/contribution_guidelines/vocabulary.rst

```diff
@@ -26,10 +31,16 @@ The following code block sketches the vocabulary implementation for a single sch
      }
    }
 
+.. note::
+
+   This file is auto-generated with each new schema commit on openMINDS. However, the attributes ``"color"``, ``"description"``, ``"label"``, and ``"semanticEquivalent"`` can be manually overwritten.
+
 Properties
 ##########
 
-The following code block sketches the vocabulary implementation for a single property:
+In openMINDS, general information on properties (e.g., description) are considered schema and (mostly) version independent. We maintain these properties information centrally on the main openMINDS GitHub repository under: `vocab/properties.json <https://github.com/openMetadataInitiative/openMINDS/blob/main/vocab/properties.json>`_
+
+This file is a simple JSON, where each vocabulary implementation for a single property is structured in the following way:
```

Comment:
see comment above, "a JSON file", not "a JSON"

Reply:
done